modules/op-pipelines-as-code-logging-configuration.adoc

```diff
@@ -0,0 +1,101 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-pipelines-as-code.adoc
+
+:_content-type: CONCEPT
+[id="pipelines-as-code-logging-configuration_{context}"]
+= {pac} logging configuration
+
+{pac} uses the `pac-config-logging` configuration map in the same namespace as the controllers. The default namespace is `pipelines-as-code`.
```

Comment:
Did you check with developers that users still can modify this configmap directly? Other configmaps for configuration are now not offered to the user, instead one should use TektonConfig, so can you ask them if this might be the case here as well?

Reply:
This cannot be modified today through the operator, and if a user does change this in cluster, it will be overwritten by the operator.

So I think we should delay documenting this until it's supported (so 1.12 🙃 )